plugins/modules/provision_workflow_manager.py

```diff
@@ -1003,31 +1076,63 @@ def reprovision_wired_device(self):
             with the reprovisioning status, task ID, and other relevant details. If an error occurs during
             the reprovisioning process, it logs the error and adjusts the status accordingly.
         """
+        if self.compare_dnac_versions(self.get_ccc_version(), "2.3.5.3") <= 0:
+            try:
```

Comment:
self.log("Starting reprovisioning of wired device using 're_provision_wired_device' API.", "DEBUG")

Reply:
updated the code